wemake_python_styleguide/violations/best_practices.py

```diff
@@ -1855,3 +1857,41 @@ class StringConstantRedefinedViolation(ASTViolation):
 
     error_template = 'Found alphabet as strings: {0}'
     code = 447
+
+
+@final
+class IncorrectExceptOrderViolation(ASTViolation):
+    """
+    Forbids the use incorrect order of ``except``.
+
+    Reasoning:
+        Using ``except Exception`` in incorrect order is error-prone.
```

Comment:
Reasoning should be: `Using incorrect order of exceptions is error-prone, since you end up with some unreachable exception clauses.`